Bottom: 1931a152223c4d743989fc31dab80dd1dbc1ea28
Top:    b13a8842d13e79ec01358d3be7e2657c853e3c0a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-22 20:35:09 +0000

hid/gtk: Disable GL alpha-blending for the pinout preview

We aren't using subcompositing for this, so don't alpha-blend.


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 7df7007..2b72efd 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1083,8 +1083,8 @@ ghid_pinout_preview_expose (GtkWidget *widget,
   ghid_start_drawing (gport, widget);
   hidgl_start_render ();
 
-  glEnable (GL_BLEND);
-  glBlendFunc (GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
+//  glEnable (GL_BLEND);
+//  glBlendFunc (GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
 
   glViewport (0, 0, allocation.width, allocation.height);
